lib/foreman/renderer/doc_templates/basic_ruby_methods.rb

```diff
@@ -472,6 +472,15 @@ module Array
 a.fetch(-1)              #=> 44
 a.fetch(4, "cat")        #=> "cat"'
           end
+
+          apipie_method :rand, 'Returns random floating point numbers between 0 and 1' do
+            optional :args, ::Float, desc: 'Can take a parameter as max value for random number. For negative and float numbers produce interesting outputs'
+            returns ::Float
+            example 'rand #=> 0.5
+rand #=> 0.8
+rand(100) #=> 23
+rand(100) #=> 72'
+          end
```

Comment:
Ruby Kernel.rand does not only return float, it does return float if the attribute is less or equal 1, otherwise it returns an integer. Please update the documentation.